conans/client/loader_parse.py

```diff
@@ -95,7 +95,7 @@ class ConanFileTextLoader(object):
     def __init__(self, input_text):
         # Prefer composition over inheritance, the __getattr__ was breaking things
         self._config_parser = ConfigParser(input_text,  ["requires", "generators", "options",
-                                                         "imports"], parse_lines=True)
+                                                         "imports", "build_requires"], parse_lines=True)
```

Comment:
This might be considered breaking in the sense that conanfile.txt that are created with this feature won't load at all in previous conan versions. But as it is a consumer file, and we can consider only forward compatibility for it, I think we are good. But I'd like to check with @lasote anyway.

Reply:
I think we are good in this case.